### PR TITLE
solved Task1 in zeynep package

### PR DESCRIPTION
**Pull request by zeynepcakm**

### CODE REVIEW

**Review by ersuamine — APPROVED** [review]
Good work

**Review by zeynepcakm — COMMENTED** [review]
I don't know how to try if my code works though

File: src/zeynep/MapBully.java
Comment (ersuamine) [inline src/zeynep/MapBully.java:1]: It looks good. 👍

**Comment by ersuamine** [comment]
> I don't know how to try if my code works though

You can make main method intelliJ and make the mapBully method static and call it from the main.

**Pull request merged**